docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,311 @@
+# Privacy First: Send Payjoin Transactions with Payjoin Dev Kit
+
+Want to enhance your Bitcoin transactions with enhanced privacy and reduced
+fees? Look no further than Payjoin and the powerful rust-payjoin crate! Payjoin
+allows you to combine your transaction with others, minimizing individual
+footprints and potentially saving on network fees.
+
+This tutorial will guide you through the exciting world of Payjoin using Rust.
+We'll delve into the rust-payjoin library, providing step-by-step instructions
+on crafting and sending your very own Payjoin transactions. Moreover, We'll
+guide you through setting up payjoin-cli, and use it a the receiver side in the
+payjoin transaction.
+
+# Bitcoin Node & Wallets
+  In order to perform a payjoin transaction, the following code will create two
+  bitcoin wallets, "sender" and "receiver" connected to bitcoin node running in
+  regtest mode.
+
+  * You need to have docker installed!
```

Comment:
installed and daemon running*

Use subheadings `##...` instead of bullets for subheadings or `**...**` bold for emphasis sparingly. I like to keep exclamation points to exclamations but that's just style.